client/my-sites/sharing/connections/service.jsx

```diff
@@ -249,18 +381,50 @@ module.exports = React.createClass( {
 				onAction={ this.performAction }
 				isConnecting={ this.state.isConnecting }
 				isRefreshing={ this.state.isRefreshing }
-				isDisconnecting={ this.state.isDisconnecting } />
+				isDisconnecting={ this.state.isDisconnecting }
+				removableConnections={ this.getRemovableConnections( this.props.service.ID ) } />
 		);
 		return (
-			<FoldableCard
-				className={ elementClass }
-				header={ header }
-				clickableHeader
-				compact
-				summary={ action }
-				expandedSummary={ action } >
-				{ content }
-			</FoldableCard>
+			<div>
+				<AccountDialog
+					isVisible={ this.state.isSelectingAccount }
+					service={ this.props.service }
+					accounts={ accounts }
+					onAccountSelected={ this.addConnection } />
+				<FoldableCard
+					className={ elementClass }
+					header={ header }
+					clickableHeader
+					compact
+					summary={ action }
+					expandedSummary={ action } >
+					{ content }
+				</FoldableCard>
+			</div>
 		);
 	}
 } );
+
+export default connect(
+	( state, { service } ) => ( {
+		availableExternalConnections: getAvailableExternalConnections( state, service.ID ),
+		isFetching: isFetchingConnections( state, getSelectedSiteId( state ) ),
+		keyringConnections: getKeyringConnectionsByName( state, service.ID ),
+		site: getSelectedSite( state ),
+		siteUserConnections: getSiteUserConnectionsForService(
+			state, getSelectedSiteId( state ), getCurrentUserId( state ), service.ID
+		),
+		user: getCurrentUser( state ),
```

Comment:
Since we're only using `user.ID`, maybe better to pass `userId: getCurrentUserId( state )` ?